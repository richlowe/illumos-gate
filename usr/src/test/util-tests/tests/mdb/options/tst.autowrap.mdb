::set +o autowrap
::typedef -c lp32
0::printf "Pack my box with five dozen liquor jugs. How razorback-jumping frogs can level six piqued gymnasts! Amazingly few discotheques provide jukeboxes.%d" int .

0::printf "%-79d%d" int . .
0::printf "%-80d%d" int . .
0::printf "%-81d%d" int . .

::set -o autowrap
0::printf "Pack my box with five dozen liquor jugs. How razorback-jumping frogs can level six piqued gymnasts! Amazingly few discotheques provide jukeboxes.%d" int .

0::printf "%-79d%d" int . .
0::printf "%-80d%d" int . .
0::printf "%-81d%d" int . .

