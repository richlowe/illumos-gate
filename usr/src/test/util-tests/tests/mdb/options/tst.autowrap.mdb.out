Pack my box with five dozen liquor jugs. How razorback-jumping frogs can level six piqued gymnasts! Amazingly few discotheques provide jukeboxes.0
0                                                                              0
0                                                                               0
0                                                                                0
Pack my box with five dozen liquor jugs. How razorback-jumping frogs can level s
ix piqued gymnasts! Amazingly few discotheques provide jukeboxes.0
0                                                                              0
0
0
0
0
